corehq/ex-submodules/fluff/pillow.py

```diff
@@ -135,19 +135,33 @@ def _get_indicator_doc_from_class_and_id(indicator_class, doc_id):
         return indicator_class(_id=indicator_id)
 
 
+class FluffPillow(ConstructedPillow):
+    def __init__(self, indicator_class, processor):
+        self.indicator_class = indicator_class
+        self.kafka_topic = indicator_class().kafka_topic
+        self.domains = processor.domains
+        self.doc_type = processor.doc_type
```

Comment:
having these as props on the pillow is useful in a few places (signals and reindexer)